### ISSUE TITLE
add a heat map

### ISSUE THREAD

**Issue by 6018203135** [Amshaker/unetr_plus_plus#66]
Excuse me, if I want to add a heat map to the btcv test, where should I add it?

**Comment by Amshaker** [comment]
Hi @6018203135 ,
What do you mean by heat map to the btcv test? Please explain what you want to do in more detail.

**Comment by 6018203135** [comment]
@Amshaker 
Yes, I'd like to use heatmaps to highlight the role of EPA (or a new module that I've trotted out myself) for visualization when testing datasets. How should I add the 3D heatmap code in your code? Thank you very much.

**Comment by Amshaker** [comment]
I attempted to visualize the 3D heatmaps within the EPA block, but encountered significant challenges due to (1) the input image size being Nx512x512, while this image is divided into patches with input size 128x128x64, and (2) there is post-processing at the end. Considering the output of the EPA or your module directly may not yield straightforward results. Instead, I suggest visualizing the final output both with and without your module. This approach provides a more accessible way to convey the impact of your module, showcasing its effects on the ultimate output rather than on intermediate features. ;)

**Comment by 6018203135** [comment]
@Amshaker 
Thank you very much!
